subgraphs/abracadabra/src/common/getters.ts

```diff
@@ -0,0 +1,150 @@
+  }
+
+  return usageMetrics;
+}
+
+export function getOrCreateMarketDailySnapshot(event: ethereum.Event): MarketDailySnapshot {
+  let id: i64 = event.block.timestamp.toI64() / SECONDS_PER_DAY;
+  let marketAddress = event.address.toHexString();
+  let marketMetrics = MarketDailySnapshot.load(marketAddress.concat("-").concat(id.toString()));
+
+  if (!marketMetrics) {
+    marketMetrics = new MarketDailySnapshot(marketAddress.concat("-").concat(id.toString()));
+    marketMetrics.protocol = BENTOBOX_ADDRESS;
+    marketMetrics.market = marketAddress;
+    marketMetrics.rewardTokenEmissionsAmount = [];
+    marketMetrics.rewardTokenEmissionsUSD = [];
+
+    marketMetrics.save();
+  }
+
+  return marketMetrics;
+}
+
+export function getOrCreateFinancials(event: ethereum.Event): FinancialsDailySnapshot {
+  // Number of days since Unix epoch
+  let id: i64 = event.block.timestamp.toI64() / SECONDS_PER_DAY;
+
+  let financialMetrics = FinancialsDailySnapshot.load(id.toString());
+
+  if (!financialMetrics) {
+    financialMetrics = new FinancialsDailySnapshot(id.toString());
+    financialMetrics.protocol = BENTOBOX_ADDRESS;
+
+    financialMetrics.feesUSD = BIGDECIMAL_ZERO;
+    financialMetrics.totalVolumeUSD = BIGDECIMAL_ZERO;
+    financialMetrics.totalValueLockedUSD = BIGDECIMAL_ZERO;
+    financialMetrics.supplySideRevenueUSD = BIGDECIMAL_ZERO;
+    financialMetrics.protocolSideRevenueUSD = BIGDECIMAL_ZERO;
+    financialMetrics.feesUSD = BIGDECIMAL_ZERO;
+    financialMetrics.protocolTreasuryUSD = BIGDECIMAL_ZERO;
+    financialMetrics.save();
+  }
+  return financialMetrics;
+}
+
+///////////////////////////
+///// Lending Specific /////
+///////////////////////////
+
+export function getOrCreateLendingProtocol(): LendingProtocol {
+  let LendingProtocolEntity = LendingProtocol.load(BENTOBOX_ADDRESS);
+  if (LendingProtocolEntity) {
+    return LendingProtocolEntity;
+  }
+  LendingProtocolEntity = new LendingProtocol(BENTOBOX_ADDRESS);
+  LendingProtocolEntity.name = "Abracadabra Money";
+  LendingProtocolEntity.slug = "abracadabra";
+  LendingProtocolEntity.schemaVersion = "1.1.0";
+  LendingProtocolEntity.subgraphVersion = "0.0.6";
+  LendingProtocolEntity.network = Network.ETHEREUM;
+  LendingProtocolEntity.type = ProtocolType.LENDING;
+  LendingProtocolEntity.totalUniqueUsers = INT_ZERO;
+  LendingProtocolEntity.totalValueLockedUSD = BIGDECIMAL_ZERO;
+  LendingProtocolEntity.lendingType = LendingType.CDP;
+  LendingProtocolEntity.save();
+  return LendingProtocolEntity;
+}
+
+export function getMarket(marketId: string): Market {
+  let market = Market.load(marketId);
+  if (market) {
+    return market;
+  }
+  return new Market("");
+}
+
+///////////////////////////
+///// Helpers /////
+///////////////////////////
+
+export function getCachedLiquidation(event: LogRepay): _LiquidationCache {
+  let cachedLiquidation = _LiquidationCache.load(
+    "_Liquidation-" +
+      event.transaction.hash.toHexString() +
+      "_" +
```

Comment:
this should be a dash not an underscore

Reply:
i have 2 variables, one if _Liquidation entity that is a helper variable for liquidations to carry over data between two separate Events that together hold all data needed for Liquidations to be handled